mlir/tools/mlir-miopen-driver/mlir-miopen-driver.cpp

```diff
@@ -319,25 +523,30 @@ static LogicalResult populateHostHarnessLogic(ModuleOp &module, OpBuilder &build
   } else if (dataType == builder.getBF16Type()) {
     memsetFuncName = "mcpuMemset4DBF16";
   }
+
   auto mcpuMemset4DFuncOp = FuncOp::create(
       builder.getUnknownLoc(), memsetFuncName,
       builder.getFunctionType(
           {fourDimUnknownSizeMemRefType, dataType}, {}));
   module.push_back(mcpuMemset4DFuncOp);
 
-  mlir::Value filterMemsetValue, inputMemsetValue, outputMemsetValue;
+  mlir::Value filterMemsetValue, inputMemsetValue, outputMemsetValue,
+      ref2MemsetValue;
```

Comment:
Why does ref2MemsetValue has to be on the interface? You should be able to allocate this within the validation routine.

Reply:
Then I need duplicate code to set it. Maybe I can extract the code that sets all MemsetValues.